FaceDetectionServer/REST-API.md

```diff
@@ -0,0 +1,240 @@
+# MobiledgeX Computer Vision Server REST API
+
+## Face Detection
+`/detector/detect/`
+
+Used to send a face image to the server and get back a set of coordinates for any detected faces.
+
+### POST
+`http://<hostname>:8008/detector/detect/`
+
+#### Parameters
+| Parameter | Description |
+| - | - |
+| image | The face(s) image. |
+
+#### Return
+JSON string with the following attributes.
+
+| Attribute | Description |
+| - | - |
+| success | "True" if face detected, otherwise "false". |
+| server_processing_time | The time in milliseconds that it took the server to process this image. |
+| rects | A JSON array of coordinates containing the detected faces. Only included if "success"="true". |
```

Comment:
We should add an explanation for the format we return of the coordinates i.e. [lowerBoundX , lowerBoundY, upperBoundX, upperBoundY]

Reply:
Oh also we should mention where (0,0) as in is the bottom left hand corner or center of the image?